theano/tensor/signal/tests/test_pool.py

```diff
@@ -1213,12 +1214,13 @@ def test_infer_shape(self):
         t_data = tensor.ftensor4()
         t_rois = tensor.fmatrix()
 
-        data = numpy.asarray(numpy.random.rand(1, 2, 32, 32), dtype=theano.config.floatX)
+        data = numpy.asarray(numpy.random.rand(1, 2, 32, 32),
+                             dtype='float32')
         rois = numpy.array([[0, 0, 0, 3, 3],
-                            [0, 0, 0, 7, 7]], dtype=theano.config.floatX)
+                            [0, 0, 0, 7, 7]], dtype='float32')
 
         self._compile_and_check([t_data, t_rois],
-                                [self.op(t_data, t_rois)],
+                                self.op(t_data, t_rois),
                                 [data, rois],
                                 self.op_class)
 
```

Comment:
@nouiz this is a duplicate of your commit to this branch and it always happens when I pull from the origin. Is there anyway I can avoid duplicating commits ?